mwdb/model/user.py

```diff
@@ -16,6 +16,13 @@
 from .object import ObjectPermission
 
 
+favorites = db.Table(
+    'favorites',
+    db.Column('user_id', db.Integer, db.ForeignKey('user.id'), index=True),
+    db.Column('object_id', db.Integer, db.ForeignKey('object.id'), index=True),
```

Comment:
As discussed, it would be nice to add the unique index for both columns
```suggestion
    db.Column('object_id', db.Integer, db.ForeignKey('object.id'), index=True),
    db.Index('ix_favorites_object_user', 'object_id', 'user_id', unique=True)
```